spec.html

```diff
@@ -1935,29 +1966,29 @@ <h1>BigIntBitwiseOp ( _op_, _x_, _y_ )</h1>
         <emu-clause id="sec-numeric-types-bigint-bitwiseAND">
           <h1>BigInt::bitwiseAND ( _x_, _y_ )</h1>
           <emu-alg>
-            1. Return BigIntBitwiseOp(`&amp;`, _x_, _y_).
+            1. Return BigIntBitwiseOp(BinaryAnd, _x_, _y_).
```

Comment:
this works and is better; but i'm not sure if it's right to pass around abstract ops as first-class values.

Reply:
There's some precedent for this. 
- The `job` parameter of `EnqueueJob` is (the name of) an abstract operation.

- The `op` parameter of `GetModifySetValueInBuffer` and `AtomicReadModifyWrite` is a "semantic function" or a "combining operation", which appears to be a kind of abstract operation.

- The RegExp runtime semantics pass around various kinds of "internal closures" (MatchResult, Continuation, Matcher, AssertionTester), which are (roughly speaking) dynamically created abstract operations.

- The `steps` parameter of `CreateBuiltinFunction` isn't a abstract operation, but it's pretty close.

- `NumberToRawBytes` binds an abstract operation to an alias, which is treating it as a first-class value.